packages/api/apollo-server/types.ts

```diff
@@ -1,13 +1,82 @@
 import { gql } from 'apollo-server-core'
 
 const Types = gql`
+  input UserInput {
+    id: String
+    firstName: String
+    lastName: String
+    utorID: String
+    userType: UserType
+  }
+
   type Query {
-    getUser: User
-    getUsers: [User]
+    getUser(id: String, utorID: String, email: String): User
```

Comment:
Should always return a User.

Reply:
> Should always return a User.

You mean change the return type from `User` to `User!`? If so, what if the user is not found?

Reply:
It should return an error with 404. I personally want to avoid ternary expressions/if statements if possible.

Reply:
> It should return an error with 404. I personally want to avoid ternary expressions/if statements if possible.

Fixed for all Query and Mutations